docs/guide/topics/embeds.rst

```diff
@@ -0,0 +1,318 @@
+We'll use :meth:`my_weather_embed.set_author() <Embed.set_author>` to set these values.
+
+.. code-block:: python3
+
+    my_weather_embed.set_author(
+        name="Today's Weather",
+        url="https://goo.gl/search/Weather+In+San+Francisco",
+    )
+
+That should look a lot like this:
+
+.. image:: /images/guide/topics/embeds/author_embed_1.png
+    :scale: 38%
+
+It appears like a "subtitle" above the title in the embed.
+
+With :attr:`Embed.author`, we can also set the icon of the author.
+The ``icon_url`` keyword-only argument of :meth:`my_weather_embed.set_author() <Embed.set_author>` accepts a string, or anything that can be cast to a string, as the URL.
+This allows us to conveniently provide an :class:`Asset` instance, which is used throughout the library.
+
+In this example, we will use the :meth:`bot.user.display_avatar <ClientUser.display_avatar>`, an :class:`Asset` instance, for the icon.
+
+.. code-block:: python3
+    :emphasize-lines: 3
+
+    my_weather_embed.set_author(
+        name="Today's Weather",
+        url="https://goo.gl/search/Weather+In+San+Francisco",
+        icon_url=bot.user.display_avatar,
+    )
+
+.. image:: /images/guide/topics/embeds/author_embed.png
+    :scale: 50%
+
+This isn't the only place you can add images though. Let's see how else we can incorporate images into our weather embed.
+
+Images
+~~~~~~~
+
+There are two ways to add images to an embed:
+
+- As the embed's ``image``.
+- As the embed's ``thumbnail``.
+
+We will add an `image of the Golden Gate Bridge`_ to the weather embed by calling :meth:`my_weather_embed.set_image() <Embed.set_image>`:
+
+.. code-block:: python3
+
+    image_url = "https://upload.wikimedia.org/wikipedia/commons/0/0c/GoldenGateBridge-001.jpg"
+    my_weather_embed.set_image(url=image_url)
+
+.. _image of the Golden Gate Bridge: https://commons.wikimedia.org/wiki/Golden_Gate_Bridge#/media/File:GoldenGateBridge-001.jpg
+
+.. image:: /images/guide/topics/embeds/image_embed.png
+    :scale: 50%
+
+As seen above, when setting :attr:`Embed.image`, the provided URL will be displayed at the bottom of the embed.
+
+The alternative to this, is to set :attr:`Embed.thumbnail`, which would be displayed in the top right corner of the embed.
+
+Rather than setting the image to a URL like we did before, let's try attaching a file for thumbnail.
+
+First, we have to construct a :class:`File` object. The first argument is the file path, and the second is the name of the attachment that will be used to refer to it within Discord.
+
+.. code-block:: python3
+
+    my_file = discord.File('./images/sunny_weather.png', 'thumbnail.png')
+
+Next, we need to call :meth:`my_weather_embed.set_thumbnail() <Embed.set_thumbnail>` to set the thumbnail.
+To refer to our attachment for the thumbnail, we will use a special URI scheme that Discord provides - ``attachment://``.
+
+Since we called our file ``thumbnail.png``, we will set the ``url`` parameter to ``attachment://thumbnail.png``.
```

Comment:
I'm a bit conflicted on whether we should explain some of the nuances of `attachment://` or not.
Filenames get pivoted by Discord before they get attached to this protocol (e.g. `my image.png` becomes `attachment://my_image.png`) so maybe it's worth touching on it? Might be kind of out of the scope, but I also don't know where else we would bring this up.